src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -52,6 +53,10 @@ public StorageOperationException(String message) {
         }
     }
 
+    public static class StorageReadOnlyException extends StorageOperationException {
+        public StorageReadOnlyException(String message)  { super(message); }
```

Comment:
[Coding standard](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) violation! One more more of these problems detected:
1. extra/missing spaces.

There is 2 spaces between your `(String message) { super(message)...;`